src/components/DropdownIcon/DropdownIcon.js

```diff
@@ -0,0 +1,114 @@
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+import { DROPDOWN_CLASSES, BUTTON_CLASSES, ICON_CLASS } from '../../constants/classes';
+import { uuid4 } from '../../utils/utils';
+
+/**
+ * @uxpincomponent
+ * @uxpinwrappers
+ */
+
+export default class DropdownIcon extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { id: uuid4() };
+  }
+
+  render() {
+    const itemsToRender = [];
+
+    Array(11).fill()
+      .forEach((_, i) => {
+        if (this.props[`item${i}`]) {
+          itemsToRender.push(
+            <a
+              className={`${DROPDOWN_CLASSES.ITEM}`}
+              href="#"
+              onClick={(e) => {
+                const currentItemActive = e.target.parentNode.querySelector('a.-active');
+
+                if (currentItemActive) currentItemActive.classList.remove('-active');
```

Comment:
```suggestion
                if (currentItemActive) currentItemActive.classList.remove(ACTIVE_CLASS);
```
From classes.js please